Sources/Core/TextEditor/UseCase/TextEditorGetVerticalSpacingUseCase.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  TextEditorGetVerticalSpacingUseCase.swift
+//  SparkTextField
+//
+//  Created by robin.lemaire on 05/09/2024.
+//  Copyright © 2023 Adevinta. All rights reserved.
+//
+
+import Foundation
+import SparkTheming
+@_spi(SI_SPI) import SparkCommon
+import SwiftUI
+
+// sourcery: AutoMockable, AutoMockTest
+protocol TextEditorGetVerticalSpacingUseCaseable {
+    // sourcery: font = "Identical"
+    func execute(height: CGFloat, font: TypographyFontToken) -> CGFloat
```

Comment:
Is it possible to add `any` before TypographyFontToken in protocols? don't remember